site/gatsby-site/src/utils/AlgoliaUpdater.js

```diff
@@ -204,16 +207,27 @@ class AlgoliaUpdater {
   uploadToAlgolia = async ({ language, entries }) => {
     const indexName = `instant_search-${language}`;
 
+    const featuredReplicaIndexName = indexName + '-featured';
+
     const index = this.algoliaClient.initIndex(indexName);
 
     await index.saveObjects(entries);
 
-    await index.setSettings({
-      ...algoliaSettings,
-      attributeForDistinct: 'incident_id',
-      indexLanguages: [language],
-      queryLanguages: [language],
-    });
+    index
+      .setSettings({
+        ...algoliaSettings,
+        attributeForDistinct: 'incident_id',
+        indexLanguages: [language],
+        queryLanguages: [language],
+        replicas: [featuredReplicaIndexName],
+      })
+      .then(() => {
+        const featuredReplicaIndex = this.algoliaClient.initIndex(featuredReplicaIndexName);
+
+        featuredReplicaIndex.setSettings({
```

Comment:
This line too is missing an `await `or  returning the promise

Reply:
Added. Does it look good now?

Reply:
Almost! Line 227 is missing an await  or a return

Reply:
woot not sure if you didn't push or  github is showing me an old diff

Reply:
It's showing me an old diff unless I go directly [the commit](https://github.com/responsible-ai-collaborative/aiid/pull/877/commits/caa08c69bf3439cfcb74d3de81dcecb68e4db000).

Reply:
that was it thanks ⚡